roles/gitea-ocp/templates/config_map.yml.j2

```diff
@@ -31,7 +31,7 @@ data:
 
     [server]
     ROOT_URL         = {{ 'http' if not _gitea_ssl|bool else 'https' }}://{{ _gitea_actual_route|default(_gitea_expose_uri) }}
-    SSH_DOMAIN       = {{ _gitea_actual_route|default(_gitea_expose_uri) }}
+    SSH_DOMAIN       = {{ gitea_service.resources[0].status.loadBalancer.ingress[0].hostname | default(_gitea_expose_uri) }}
```

Comment:
Is there a chance this will spit out failures in the event that it is _not_ exposed?